docs/_embed/solutions/partner-api.md

```diff
@@ -349,14 +352,50 @@ Pragma: no-cache
   "refresh_token": "9D9oz2ZzaouT12LpvklQwNBf6s4vva"
 }
 ```
-🎉 This response contains the access token that you'll use to make API request on the user's behalf.
+🎉 This response contains the access token that you'll use to make API request on the user's behalf, as well as a
+refresh token which should be stored securely for later use.
 
 ##### 5. Using the access token
-The access token should be passed with requests an `Authorization` header. For example:
+The access token should be passed with requests as an `Authorization` header. For example:
 ```
 Authorization: Bearer jk8s9dGJK39JKD93jkd03JD
 ```
 
+##### 6. Refreshing the access token
+All access tokens will expire after 10 hours (the number of seconds in `expires_in`), for security purposes. After
+that point, any request using that access token will return a 401 status code. To proceed, the refresh token should be
+exchanged for a new access token _and_ a new refresh token. This will not require any interaction by the user.
+
+Below is an example request that can be used:
+```cURL
```

Comment:
#question Does curl get special syntax highlighting here? Otherwise I would've expected `sh` as the file type

Reply:
Good catch! I copied `cURL` from above, but it doesn't have any special syntax highlighting here or in Mintlify (and looks rather bad [here](https://platform.zapier.com/embed/partner-api#4-exchange-authorization-code-for-an-access-token)). `sh` is better :) will fix!